### PR TITLE
api/setup

### PR DESCRIPTION
**Pull request by melindabernrdo**
-[x] Sets up basic server and `/status` endpoint
-[x] Adds README.md 
-[x] Adds migration package 
  -[x] Create user tables migration
  -[x] Create cards tables migration

### CODE REVIEW

File: db/migrations/00001_user_tables.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	// create account types enum
+	_, err := tx.Exec(`CREATE TYPE account_type AS ENUM ('basic')`)
+	if err != nil {
+		return err
+	}
+
+	// create account types table
+	_, err = tx.Exec(`CREATE TABLE IF NOT EXISTS account_types (
+			id SERIAL PRIMARY KEY, 
+			type account_type DEFAULT 'basic',
```
Comment (c9rl) [inline db/migrations/00001_user_tables.go:23]: ENUM?

Comment (c9rl) [reply db/migrations/00001_user_tables.go:23]: w. `basic` and `admin`

File: db/migrations/00002_create_cards_tables.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	   )
+	`)
+	if err != nil {
+		return err
+	}
+
+	// create black cards table
+	_, err = tx.Exec(`CREATE TABLE IF NOT EXISTS black_cards (
+			id SERIAL PRIMARY KEY, 
+			text VARCHAR(200) NOT NULL,
```
Comment (c9rl) [inline db/migrations/00002_create_cards_tables.go]: Can we use a text field here? Idk if I want to limit it to 200 chars.

File: db/migrations/00002_create_cards_tables.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		   )
+		`)
+	if err != nil {
+		return err
+	}
+
+	// create white cards table
+	_, err = tx.Exec(`CREATE TABLE IF NOT EXISTS white_cards (
+		id SERIAL PRIMARY KEY, 
+		text VARCHAR(200) NOT NULL,
```
Comment (c9rl) [inline db/migrations/00002_create_cards_tables.go]: Same.

**Review by c9rl — CHANGES_REQUESTED** [review]
Some minor questions/notes.

File: db/migrations/00001_user_tables.go (hunk 1 above)
Comment (melindabernrdo) [reply db/migrations/00001_user_tables.go:23]: Postgres syntax requires you to declare the enum outside of the column declaration. The statement above creates the enum:
```_, err := tx.Exec(`CREATE TYPE account_type AS ENUM ('basic', 'admin')`)```

**Comment by c9rl** [comment]
2harry2potter

**Review by c9rl — APPROVED** [review]

**Pull request merged**